### DISCUSSION TITLE
How do I get line breaks back in the userscript?

### DISCUSSION THREAD

**Discussion by gaveitatry2** [violentmonkey/violentmonkey#1592] [Q&A]
I used to use various number of line breaks between sections of code (two line breaks, three line breaks, five line breaks) but I am no longer able to do that.  As soon as I save, it goes back to a maximum of one line break.  How do I get he old behavior back?

**Comment by tophf** [comment]
It's a bug, sorry about that. The workaround until the next version is released is to add `"killTrailingSpaceOnSave": false,` in the editor options in Violentmonkey settings:

![image](https://user-images.githubusercontent.com/1310400/193199103-cf15abab-e83b-401d-af69-62136fb0ee55.png)
